src/exception.cr

```diff
@@ -17,6 +17,8 @@ class Exception
   # This is useful for wrapping exceptions and retaining the original
   # exception information.
   getter cause : Exception?
+
+  # :nodoc:
   property callstack : CallStack?
```

Comment:
Why make this nodoc? This seems like an important component of the API.
It should be the other way around: Document `Exception::CallStack`.

Reply:
I think is rather a runtime internal. The main justification to move CallStack inside Exception for me is that it was `:nodoc:` at a first place.

Reply:
I agree. This was an undocumented type. The idea seems to be to hide it more, not to suddenly expose it. That can always happen later, though.

Reply:
Oh, never mind. I confused it with `backtrace`. Similar story as above...